components/TestimonialLists/index.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <div>
+    <Card :testimonials="testimonials" />
+  </div>
+</template>
+
+<script>
+
```

Comment:
Minor: 

inconsistency, after tag `script` you are not added a new line but in this section you add new line.